target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -253,6 +253,14 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth1"
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:lan" "eth0"
```

Comment:
You are using "$boardname:green:lan" for wan here.
Despite, the default case you are using in 02_network is `ucidef_set_interfaces_lan_wan "eth0" "eth1"`, so either LEDs or network modes are set up the wrong way.